scripts/zkai_d64_external_adapter_surface_probe.py

```
@@ -0,0 +1,550 @@
+        raise D64ExternalAdapterProbeError("decision mismatch")
+    _validate_generated_at(payload.get("generated_at"))
+    git_commit = payload.get("git_commit")
+    if not isinstance(git_commit, str) or not git_commit:
+        raise D64ExternalAdapterProbeError("git_commit must be a non-empty string")
+
+    fixture = FIXTURE.build_fixture()
+    if payload["target"] != fixture["target"]:
+        raise D64ExternalAdapterProbeError("target drift")
+    source = payload["source_fixture"]
+    if source.get("statement_commitment") != fixture["statement"]["statement_commitment"]:
+        raise D64ExternalAdapterProbeError("source statement commitment drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)